src/routes/InventoryTable.js

```diff
@@ -157,6 +163,8 @@ const Inventory = ({
             });
         });
         dispatch(actions.clearNotifications());
+        //we have to fetch groups to make them available in state
+        dispatch(fetchGroups());
```

Comment:
I'd vote for not having this call here on the very top of the table component, but rather put it to SearchInput.js in useEffect. This way, you will make sure the data for the groups list is fetched when the modal is open.

This version can also work but we have to strive for not further overwhelming the high-level component and make separate components more independent.

Reply:
Moved it to the AddHostToGroup